generator/src/main/java/com/scottlogic/deg/generator/App.java

```diff
@@ -9,7 +10,7 @@
 public class App implements Runnable {
     private static final CommandLine picoCliCommandLine = new CommandLine(new App())
         .addSubcommand("generate", new GenerateCommandLine())
-        .addSubcommand("generateTestCases", new GenerateTestCases())
+        .addSubcommand("generateTestCases", new GenerateTestCasesCommandLine())
```

Comment:
Is now a good time to rename this from generateTestCases to violate

Reply:
Lets consider it as a separate change, there is benefit in this PR being focused on its change rather than changing documentation and verbs, etc.